### ISSUE TITLE
Illegal tab character in putty.sls

### ISSUE THREAD

**Issue by sbuller** [saltstack/salt-winrepo-ng#964]
`sudo salt -G 'os:windows' pkg.refresh_db` yields the following (elided) error message.

```
    ERROR: Error occurred while generating repo db. Additional info follows:
    
    failed:
        1
    failed_list:
        ----------
        salt-winrepo-ng\putty.sls:
            - Failed to compile 'salt-winrepo-ng\putty.sls': Illegal tab character; line 8

```

**Comment by TheBigBear** [comment]
@sbuller thanks for reporting this issue, sorry about this. 

@mercsniper did you not test the new putty file? It looks like it was failing on whitespace/identation. Appears you accidentally used at least a couple TABs. Do I need to revert your change and go and test it myself, or can you test this (hopefully) corrected version quite soon?

**Comment by sbuller** [comment]
no problem, the error message was helpful in getting things working in the interim.

**Comment by TheBigBear** [comment]
fixed in merge #965
